packages/core/src/generators/utils/prompt-for-template.ts

```diff
@@ -0,0 +1,35 @@
+import { DotNetClient } from '@nx-dotnet/dotnet';
+
+import { prompt } from 'inquirer';
+
+export function promptForTemplate(
+  client: DotNetClient,
+  search?: string,
+  language?: string,
+): string | Promise<string> {
+  const available = client.listInstalledTemplates({ search, language });
+
+  if (available.length === 1) {
+    return available[0].shortNames[0];
```

Comment:
This will be unexpected behavior for most users. A user who supplies no template option, but only has one installed template, will expect a list of templates to choose from (even though only one choice is available). A user who supplies a template option, but only has one installed template, will expect their specified template to be the one chosen, even though it's not installed. It's incredibly unlikely that a user only has one template (since many ship with .NET itself), but this is still unusual behavior.